src/panes/transactions/tx-details.tsx

```diff
@@ -18,10 +22,27 @@ export const TxDetails: React.FC<TxDetailsProps> = ({ txData, isFocused }) => {
       }
     }
   }
+  const ref = useRef<blessed.Widgets.BoxElement>(null)
+  useEffect(() => {
```

Comment:
feels like we should make a ticket to handle focus for everything in a higher-level component so we don't have to do this in every child

Reply:
the problem is you need a reference to the element in order to focus it

Reply:
think this problem goes away when we leave react?

Reply:
nope, you still have to call `.focus()` on the actual element that wants to be focused regardless

Reply:
but there will be a lot less boilerplate without react